imap/imapd.c

```diff
@@ -7780,7 +7780,13 @@ static void cmd_rename(char *tag, char *oldname, char *newname, char *location)
             state = imapd_index;
         }
         else {
-            r = index_open(oldmailboxname, NULL, &state);
+            struct index_init init;
+            memset(&init, 0, sizeof(init));
```

Comment:
this could be `struct index_init init = {0};`

Reply:
Yep.  Will fix.  We just discussed this over dinner.  Will ad this to coding standards

Reply:
This feels very amenable to a test too!

Reply:
Initialization fixed